src/api/vbucket.cc

```diff
@@ -12,12 +16,26 @@ Status VBucket::Link(std::string blob_name, std::string bucket_name,
   (void)ctx;
   Status ret = 0;
 
-  LOG(INFO) << "Linking blob "<< blob_name << " in bucket "
-            << bucket_name << " to VBucket " << name_ << '\n';
-
-  // inserting value by insert function
-  linked_blobs_.push_back(make_pair(bucket_name, blob_name));
-
+  LOG(INFO) << "Linking blob " << blob_name << " in bucket " << bucket_name
+            << " to VBucket " << name_ << '\n';
+
+  // FIXME(hari): to use both bucket name and blob name
```

Comment:
Is this `FIXME` still relevant since you implemented `BucketContainsBlob`?

Reply:
No it is not needed.